packages/blockchain-wallet-v4-frontend/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { ComponentMeta } from '@storybook/react'
+
+import { Tab, Tabs, TabsComponent } from '.'
+
+const tabsStoriesMeta: ComponentMeta<TabsComponent> = {
+  component: Tabs,
+  subcomponents: {
+    Tab
+  },
+  title: 'Components/Tabs'
+}
+
+export const Default = () => {
+  const [selectedTab, setSelectedTab] = useState<
+    'all' | 'live' | 'last_day' | 'last_week' | 'last_month' | 'last_year'
+  >('live')
+
+  return (
+    <Tabs>
+      <Tab
+        badgeColor='green'
+        onClick={() => setSelectedTab('live')}
+        selected={selectedTab === 'live'}
+      >
+        Live
```

Comment:
We'll need to convert these to formatted messages for translations but it's fine like this for now.

Reply:
ah yes, when we actually use this components we can do this
```
<Tabs>
  <Tab>
    <ForemttedMessage />
  </Tab>
</Tabs>
```